docs/service-response-examples/get-events.json

```diff
@@ -0,0 +1,590 @@
+[
```

Comment:
Is this helpful to keep around?

Reply:
Eh, it's your repo. I can remove it if you want. But nice to have a reference without having to load the site and view the network tab